test/parallel/test-timers-max-duration-warning.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const timers = require('timers');
+
+const TIMEOUT_MAX = 2147483647; // 2^31-1
```

Comment:
Nit: `Math.pow(2, 31) - 1`? Not sure that's any better. Feel free to ignore. Your choice.